enterprise/frontend/src/embedding-sdk/cli/snippets/theme-switcher-snippet.ts

```diff
@@ -23,7 +23,7 @@ const ICONS = {
     <svg viewBox="0 0 24 24">
       <path
         fill="none"
-        stroke="currentColor"
+        stroke="#2D353A"
```

Comment:
restores the color of the theme switcher. it was showing as white, so it seems to disappear as it blends with the white background.